src/rules/prefer-lodash-method.js

```diff
@@ -88,7 +105,11 @@ module.exports = {
             }),
             'CallExpression:exit'(node) {
                 if (!usingLodash.has(node) && !shouldIgnore(node) && canUseLodash(node)) {
-                    context.report(node, `Prefer '_.${getMethodName(node)}' over the native function.`)
+                    let lodashMethodName = getMethodName(node)
+                    if (isNativeStringMethodCall(node)) {
+                      lodashMethodName = get(nativeStringMap, lodashMethodName)
```

Comment:
Good call.  I was originally going to do that, but figured it was a lodash lib, so I'd use a lodash function :)  Should get better performance by making the change you suggested!